### ISSUE TITLE
https://github.com/hashicorp/vagrant/issues/11139

### ISSUE THREAD

**Issue by copremesis** [hashicorp/vagrant#11664]
https://github.com/hashicorp/vagrant/issues/11139

Please open this issue it's still happening

no idea why it's closed. Thank you

**Comment by briancain** [comment]
Hey there @copremesis - Please open a new issue and fill out the issue template provided and give us a debug log, the expected versus actual behavior you are experiencing, the version of Vagrant you are using, and a Vagrantfile so that we can help you. I'm going to go ahead and close this issue since those details were left out.

**Comment by copremesis** [comment]
sigh ... why not just open the one that's still broken?

**Comment by copremesis** [comment]
no point in repeating what's already in place ... please just open the ticket again since we're seeing this in version 2.6

**Comment by copremesis** [comment]
https://github.com/hashicorp/vagrant/issues/11139 ( details are in this close one )

**Comment by briancain** [comment]
That issue has already been fixed via: https://github.com/hashicorp/vagrant/pull/11152

Omitting the guest name no longer attempts to save a snapshot for a guest with the passed in snapshot name. That issue has been resolved and does not exist in the newest version of Vagrant.

```
brian@localghost:vagrant-sandbox % be vagrant snapshot save foo                                                                                                                                                                  ±[●●][master]
Machine 'test' has not been created yet, and therefore cannot save snapshots. Skipping...
==> hashicorp: Snapshotting the machine as 'foo'...
==> hashicorp: Snapshot saved! You can restore the snapshot at any time by
==> hashicorp: using `vagrant snapshot restore`. You can delete it using
==> hashicorp: `vagrant snapshot delete`.
Machine 'vagrant-share' has not been created yet, and therefore cannot save snapshots. Skipping...
Machine 'vbox' has not been created yet, and therefore cannot save snapshots. Skipping...
Machine 'libvirt' has not been created yet, and therefore cannot save snapshots. Skipping...
This provider doesn't support snapshots.

This may be intentional or this may be a bug. If this provider
should support snapshots, then please report this as a bug to the
maintainer of the provider.
```

I don't actually know what issue you are running into without seeing a debug log and an expected and actual behavior description from your system. If you are saying your details are identical to https://github.com/hashicorp/vagrant/issues/11139, then it has already been fixed as of version 2.2.7 and this issue is just a duplicate of a resolved issue. Please upgrade to a version of Vagrant 2.2.7 or newer and the fix will be included.
